casual/casual-api/src/main/java/se/laz/casual/config/Inbound.java

```diff
@@ -13,13 +13,17 @@
 {
     public static final String CASUAL_INBOUND_STARTUP_MODE = "CASUAL_INBOUND_STARTUP_MODE";
     public static final String CASUAL_INBOUND_USE_EPOLL = "CASUAL_INBOUND_USE_EPOLL";
+    public static final String CASUAL_INBOUND_INITIAL_DELAY_ENV_NAME = "CASUAL_INBOUND_INITIAL_DELAY";
```

Comment:
`CASUAL_INBOUND_STARTUP_DELAY` or `CASUAL_INBOUND_STARTUP_INITIAL_DELAY` - to be consistent with existing `CASUAL_INBOUND_STARTUP_MODE`